go/libraries/utils/argparser/args_test.go

```diff
@@ -112,23 +112,23 @@ func TestParsing(t *testing.T) {
 			expectedArgs: []string{"value"},
 		},
 		{
-			name:         "-mvalue",
+			name:         "-m value",
 			options:      []*Option{forceOpt, messageOpt},
-			args:         []string{"b", "-mvalue", "c"},
+			args:         []string{"b", "-m value", "c"},
```

Comment:
Still being able to support `-mvalue` would be a nice feature to have, and would be consistent with Git's arg parsing behavior (e.g. `git init -bfoo` and `git init -b foo` are both valid).

Reply:
Ah I see, I think I misunderstood what you said earlier. I'll try and make the appropriate changes.

Reply:
Ah my bad, I seem to have reversed the behavior for short and long form flags.

Reply:
No worries at all! Apologies if I didn't describe it clearly enough.